tests/general/LineCountBearTest.py

```diff
@@ -16,12 +16,26 @@ def setUp(self):
     def test_run(self):
         self.section.append(Setting('max_lines_per_file', 1))
         self.check_results(
-            self.uut, ['line 1', 'line 2', 'line 3'],
+            self.uut, ['line 1', 'line 2', 'line 3', '\n'],
             [Result.from_values('LineCountBear',
-                                'This file had 3 lines, which is 2 lines more '
+                                'This file had 4 lines, which is 3 lines more '
```

Comment:
I would suggest not to modify the existing tests. Just adding an additional test should be fine. :)